<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>401</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 26pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>스마트 스토어 &quot;밈석골&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>371</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(&quot;C:/Users/aaaa/Desktop/7번재 프로젝트 스마트 스토어 만들기/밈석골 돼지국밥.jpg&quot;)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="order_manage">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>400</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>주문 관리</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventory_manage">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>400</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>재고 관리</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>280</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_btn">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>280</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QPushButton" name="order">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>510</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>주문 하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>380</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="userId">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>50</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>219</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pw">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>300</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_home_1">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>110</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 확인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>310</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>연락처</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Join_name">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_id">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>170</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pw">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pw_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>270</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_call">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>320</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="join">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>470</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>470</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>주소</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_addredss">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>370</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="join_double_check">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>중복 체크</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_home_1">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>주문관리</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="order_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>1111</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상품</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>개수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>가격</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문상태</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="order_home_2">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inquire">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>70</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>조회하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="take_over">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>70</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>접수</string>
     </property>
    </widget>
    <widget class="QPushButton" name="complete">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>70</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>완료</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>재고관리</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="inquire_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>1111</width>
       <height>401</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>제작 가능 개수</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>메뉴</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>월계수 잎</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>양파</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>마늘</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>대파</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>청양고추</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>부추</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>생강</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>다진 마늘</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>멸치 액젓</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>후추</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>사골육수</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>고춧가루</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>간장</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매실액</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>돼지국밥</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>순대국밥</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>머리국밥</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>내장국밥</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="inquire_home_3">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inquire_2">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>70</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>조회하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>주문하기</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="menu1count">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>115</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="menu2count">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>205</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="menu3count">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>295</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="menu4count">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>385</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_order">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>500</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>주문하기</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="menu1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>돼지국밥</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="menu2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>머리국밥</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="menu3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>순대국밥</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="menu4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Arial&quot;;
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>내장국밥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inquire_home_4">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
